### DISCUSSION TITLE
Support for Rock 5A

### DISCUSSION THREAD

**Discussion by exceeded** [tesla-android/issue-tracker#239] [Q&A] [answered]
Recently wondering if it's possible to install this image on other SoC.

Notably the Rock 5A.

Thinking of taking the plunge and seeing if I can get it to work.

**Comment by mikegapinski** [comment, marked as answer]
It will never work. Bringing up another board would take a few months of my work and the results won't be amazing. Drivers are not fully open for those Chinese boards. I own a bunch of them, they are a complete cluster**** even on Linux.

**Reply by exceeded** [reply]
Thanks for the quick response. I'll give that a miss then!! Love what you're doing with this project. I've got mine running brilliantly with a nice fast microsd card so it's nice and snappy!
